SCF-Session-8/zooManagement/BirdZone.java

```diff
@@ -0,0 +1,13 @@
+package zooManagement;
+
+/**
+ * This class inherits the Zone class and used to add Bird Zone in the Zoo
+ * @author Ishita_Mathur
+ */
+public class BirdZone extends Zone
```

Comment:
You can remove this class `BirdZone` as it does not have any extra feature for zone. Zone can be identified by `AnimalFamily`.